documentation/releaseNotes/releaseNotes.md

```diff
@@ -1,6 +1,5 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
-- ⚠️ [Here is a breaking change we did and its work item] (#737)
-- [Here is a new feature we added and its work item] (#737)
-
-\*⚠️ **_indicates a breaking change_**
\ No newline at end of file
+- Delete endpoint on startup (#2178)
+- Fix duplicated values for `/metrics` (#2149)
+- Add `CollectLiveMetrics` collection rule action (#2119)
```

Comment:
Let's add in the fixes in #2149 and #2165, something like:

```
- Fixed memory leak in authentication handler (#2165)
- Fixed duplication of metrics (#2149)
```

and to the 6.2.2 release (with its corresponding PR numbers)

Reply:
I already had #2149 in there, but I switched the wording to what you put and added in the other one (I wasn't sure we were going to mention that in the release notes)